google/cloud/pubsub_v1/subscriber/futures.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, manager):
         super(StreamingPullFuture, self).__init__()
         self._manager = manager
```

Comment:
What about manager?

Reply:
Uhm, what about it? :)
We need a reference to it to shut the stream down if a future gets cancelled.

If the question is about the concurrency model,  StreamingPullManager accepts a `scheduler` parameter (and uses a thread-based scheduler to dispatch messages to callbacks), but the class itself relies on threading synchronization primitives internally (non-configurable).

Reply:
Why not `__manager`?

Reply:
Oh, that... tunnel vision I guess, since that's an old already existing attribute. Will rename it.